LLama/Native/LLamaModelParams.cs

```diff
@@ -13,11 +13,12 @@ public unsafe struct LLamaModelParams
         /// todo: add support for llama_model_params.devices
         /// </summary>
         private IntPtr devices;
-
-        // NULL-terminated list of buffer types to use for tensors that match a pattern
-        // actual type: llama_model_tensor_buft_override* 
-        // todo: add support for tensor_buft_overrides
-        private IntPtr tensor_buft_overrides;
+
+        /// <summary>
+        /// NULL-terminated list of buffer types to use for tensors that match a pattern
+        /// actual type: llama_model_tensor_buft_override*
+        /// </summary>
+        public IntPtr tensor_buft_overrides;

```

Comment:
```suggestion
        /// </summary>
        public LLamaModelTensorBufferOverride* tensor_buft_overrides;
```

The struct is `unsafe`, so you can just use the type directly (unless there's some specific reason not to)